.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "editor.formatOnSave": true,
+    "editor.formatOnSave": false,
```

Comment:
it should be fixed by another PR

Reply:
Nope, it shouldn't be in repo in general

Reply:
https://github.com/SAP/fundamental-ngx/pull/3768/files#r520124796

Reply:
Removed